extensions/amazon-lambda/deployment/src/main/java/io/quarkus/amazon/lambda/deployment/AmazonLambdaProcessor.java

```diff
@@ -50,6 +51,7 @@
 
     private static final DotName REQUEST_HANDLER = DotName.createSimple(RequestHandler.class.getName());
     private static final DotName REQUEST_STREAM_HANDLER = DotName.createSimple(RequestStreamHandler.class.getName());
+    private static final DotName SKILL_STREAM_HANDLER = DotName.createSimple(SkillStreamHandler.class.getName());
```

Comment:
You should use the class name as a string instead of a class. This way, you won't need the dependency if not required.

Reply:
Ok, yes, a far simpler way to do it.

That's done.  Validated against my sample Alexa project.